### ISSUE TITLE
About Taiwan,China

### ISSUE THREAD

**Issue by LimeZky** [datasets/country-codes#70]
Hello,When I read this data,I found that Taiwan,China（台湾，中国） don't appear in this data.Taiwan is a province of China,please add this into data,thank you.

**Comment by fleiber** [comment]
Taiwan does appear... but with most of columns empty (whereas the currency TWD is properly listed in currency-codes). That's indeed an issue, it's a pretty large country :)

**Comment by andrewintw** [comment]
> Hello,When I read this data,I found that Taiwan,China（台湾，中国） don't appear in this data.Taiwan is a province of China,please add this into data,thank you.

No, Taiwan actually is not part of China. Does China really govern Taiwan? Do Taiwan people pay any tax to China? Does Taiwan use the same constitution with China? Does Taiwan use the same currency-codes with China? All the answer is NO. China people always like to deceive themself.

**Comment by fleiber** [comment]
Obviously yes, it's not part of China. PRC does have enough power now to force through threats most countries to cut official ties with Taiwan, but that doesn't yet make their dreams come true :p
Anyway, that's partly why having most of the Taiwan columns empty is a bit awkward :)

**Comment by kongxianghe1234** [comment]
The People's Liberation Army will tell you everything.
shame on you

**Comment by olayway** [comment]
Closing as fixed. Taiwan is already present in the dataset as ISO 3166-1 Alpha-3 code TWN, with full data including currency (TWD) and other fields.
